notebooks/animate-anyone/animate-anyone.ipynb

```diff
@@ -0,0 +1,1349 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        ov_image_encoder = nncf.compress_weights(ov_image_encoder)
</pre><p>nncf is not defined</p><p>Add some description and link on NNCF Compression Weights documentation page</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done</p>

Reply:
<p>still not see any relevant explanations...</p>